src/main/java/gomedic/model/activity/ActivityId.java

```diff
@@ -24,6 +24,16 @@ public ActivityId(Integer id) {
         AppUtil.checkArgument(isValidActivityId(this), MESSAGE_CONSTRAINTS);
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @param id Integer from 1 to 999
```

Comment:
I think we could change param description since the param doesnt follow the `Integer from 1 to 999` format?
```suggestion
     * @param id a string of format "PDDD", where "P" is an alphabetic character and "D" is a decimal number
```

Reply:
Sure